### PR TITLE
Update tutorials overview for test network

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: NIKHIL E GUPTA <negupta@us.ibm.com>

#### Type of change

- Documentation update

#### Description

Update tutorial page overview to reflect new tutorials and deprecation of BYFN

### CODE REVIEW

File: docs/source/tutorials.rst [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+SDKs to invoke smart contracts from your client applications. For an in depth
+overview of how Fabric applications and smart contracts work together, you
+can visit the :doc:`developapps/developing_applications` topic.
+
+The :doc:`deploy_chaincode` tutorial can also be used by network operators to learn
+how to use the Fabric chaincode lifecycle to manage smart contracts deployed on
+a running network. Both network operators and application developers can use the
+tutorials on `Private data <./private_data_tutorial.html>`_ and `CouchDB <./couchdb_tutorial.html>`_
+to explore important Fabric features. When you are ready to deploy hyperledger
+Fabric in production, see the guide for :doc:`deployment_guide_overview`.
```
Comment (pamandrejko) [inline docs/source/tutorials.rst]: Hyperledger

**Review by pamandrejko — CHANGES_REQUESTED** [review]

File: docs/source/tutorials.rst (hunk 1 above)
Comment (nikhil550) [reply docs/source/tutorials.rst]: done.

**Comment by nikhil550** [comment]
Responding to comments that the test net doc should be in the tutorials as well, to make things a bit easier to find

**Review by pamandrejko — APPROVED** [review]
Good to see this in the Tutorial toc now

**Comment by pamandrejko** [comment]
@Mergifyio backport release-2.0

**Comment by nikhil550** [comment]
@Mergifyio backport release-2.1

**Pull request merged**